src/theory/sets/theory_sets_private.cpp

```diff
@@ -390,6 +390,13 @@ void TheorySetsPrivate::fullEffortCheck()
     {
       continue;
     }
+    // check set.all rules
+    checkAll();
```

Comment:
This is a bit of a misleading name, can you rename this to something that doesn't imply all things are being checked rather it is for `set.all`?

Reply:
I was hoping for a suggestion for a better name. 
How about checkPredicateAll?

Reply:
Sure

Reply:
already renamed.